server/tests/clients/test_google.py

```diff
@@ -124,3 +128,101 @@ def test_get_user_info_happy() -> None:
     assert userInfo["email"] == myEmail
     assert userInfo["id"] == myId
     assert userInfo["picture"] == myPicture
+
+
+def test_get_google_person_list_happy() -> None:
+    """Test happy path for get google person list."""
+    discovery_mock = Path(
+        os.path.join(os.path.dirname(__file__), "oauth2-discovery.json")
```

Comment:
Honestly surprised this works. 
The discovery mock should be what the rest api returns when the we  ask google "what options do I have for this endpoint".

At the top of this file theres a comment for where to find the discovery-mock data.

For the people api, you can find it at "https://people.googleapis.com/$discovery/rest?version=v1".

Please grab that file and add it to the commit/ replace the 'oauth2-discovery.json' with it.

Reply:
i made a people-discovery.json file for it, kept oauth2-discovery as well since that's needed for token tests